src/averageAges.js

```diff
@@ -15,11 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
+
+  if (arguments.length > 1) {
```

Comment:
```suggestion
  if (arguments.length > 1) {
```
В умові сказано
> replace `if ()` statement with &&, || or ?:

Давай спробуємо не використовувати іфи і замінимо це на тенарники та логічні оператори